package.json

```diff
@@ -23,7 +23,7 @@
     "lint:linkcheck": "SKIP_OG=true pnpm build && tsm --require=./scripts/lib/filter-warnings.cjs ./scripts/lint-linkcheck.ts",
     "lint:linkcheck:nobuild": "tsm --require=./scripts/lib/filter-warnings.cjs ./scripts/lint-linkcheck.ts",
     "lint:slugcheck": "node ./scripts/lint-slugcheck.mjs",
-    "lint:eslint": "eslint . --ext .js,.ts,.astro",
```

Comment:
The reason I removed the `--ext`, see https://eslint.org/docs/latest/use/command-line-interface#--ext

![](https://user-images.githubusercontent.com/1091472/212910817-708c7af3-9089-4d58-87c3-db184eafe345.png)